coop_memberspace/views/website/website_homepage.xml

```diff
@@ -49,15 +50,15 @@
                             </div>
                         </div>
                         <div class="col-md-4">
-                            <h3>Coopérateurs actifs</h3>
+                            <h3>Active cooperators</h3>
                             <div class="text-center" style="border: 1px solid #b2b7bb;padding: 10px;">
                                 <p t-esc="'{0:,.0f}'.format(num_of_members).replace(',', ' ')" />
                                 <i class="material-icons" style="font-size: 3.5em">accessibility_new</i>
                             </div>
                         </div>
                         <div class="col-md-4">
                             <div>
-                                <h3>Semaine actuelle</h3>
+                                <h3>Current week</h3>
```

Comment:
I like the fact that we're removing french terms from the code.

But we need to add them to the translation file so that, in french, they maintain their terms